app/javascript/data/advancedFilterOptions.js

```diff
@@ -178,11 +176,7 @@ export const advancedFilterOptions = [
     title: 'Primary Language (Select)',
     description: 'Monitoree primary language',
     type: 'select',
-    options: supportedLanguages.languages
-      .map(lang => {
-        return lang.name;
-      })
-      .concat(['']),
+    options: [] // calculated asynchronously in the AdvancedFilter componentt,
```

Comment:
Small typo on `component`. I wouldn't necessarily call it "calculated", it's more "populated"

Reply:
whoops. blame the broken keyboard 😭. Fixing now...